shared/js/intl_helper.js

```diff
@@ -125,7 +158,7 @@
     _fireObservers: function(affectedObjects, ...args) {
       const affectedCallbacks = new Set();
 
-      affectedObjects.forEach((obj, name) => {
+      affectedObjects.forEach(obj => {
```

Comment:
Nit (in a few other places too):  you don't need the curly braces in arrow function if there's only one expressions in the body.